Assets/MRTK/SDK/Editor/Inspectors/UX/InteractiveElement/TrackedStatesInspector.cs

```diff
@@ -0,0 +1,287 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.UI.Interaction;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Editor
+{
+    /// <summary>
+    /// Custom inspector for the Tracked States scriptable object that is contained in an object that inherits from 
+    /// BaseInteractiveElement.
+    /// </summary>
+    [CustomEditor(typeof(TrackedStates))]
+    public class TrackedStatesInspector : UnityEditor.Editor
+    {
+        private TrackedStates instance;
+        private SerializedProperty stateList;
+
+        private static GUIContent RemoveStateButtonLabel;
+        private static GUIContent AddStateButtonLabel;
+
+        // Used to set the default name for the text input field when a user creates a new state
+        private static string newStateName = "New State Name";
+
+        private const string newCoreStateName = "New Core State";
+        private const string createNewStateName = "Create New State";
+        private const string defaultStateName = "Default";
```

Comment:
Nit: It seems that these fields should be using PascalCase per https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html